signup/templates/signup/index.html

```diff
@@ -6,7 +6,7 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-6">
-            <h2>Sign Up for an Account</h2>
+            <h2>Sign Up for an Account&emsp;&emsp;&emsp;
```

Comment:
last comment: please remove those `&emsp;&emsp;&emsp;` and put the `</h2>` tag back. I'm not sure what those are but those are not valid HTML

Reply:
Done